TTHAnalysis/python/plotter/functions.cc

```diff
@@ -191,8 +191,347 @@ float DPhi_CMLep_Zboost(float l_pt, float l_eta, float l_phi, float l_M, float l
   return deltaPhi(l1.Phi(),Z.Phi());
 }
 
+// May 2016: This is func which takes the 4vectors of MET, L1,L2 and reconstructs 2 taus and returns the effective Z(-->tautau) mass.                                                                                       
```

Comment:
Blanked the file taking it from the current ECOP 80X_M17 head, as agreed on skype.